src/status_im/ui/screens/chat/message/message.cljs

```diff
@@ -130,13 +130,16 @@
            destination])
 
     "mention"
-    (conj acc [react/text-class
-               {:style    {:color (cond
-                                    (= content-type constants/content-type-system-text) colors/black
-                                    :else                                               colors/mention-incoming)}
-                :on-press (when-not (= content-type constants/content-type-system-text)
-                            #(re-frame/dispatch [:chat.ui/show-profile literal]))}
-               [mention-element literal]])
+    (conj acc
+          [react/view {:style {:background-color "rgba(67,96,223,0.1)" :border-radius 6 :padding-horizontal 3}}
+           [react/text-class
+            {:style    {:color (cond
+                                 (= content-type constants/content-type-system-text) colors/black
+                                 :else                                               colors/mention-incoming)
+                        :font-weight (cond (= content-type constants/content-type-system-text) "400" :else "500")}
```

Comment:
why cond and not if ?

Reply:
The `color` property was already using `cond` so I thought would also use `cond` for `font-weight`. Shall I change it `if`?

Reply:
yeah better to use `if`

Reply:
probably `cond` was used by mistake